daml-lf/encoder/BUILD.bazel

```
@@ -72,10 +82,37 @@ da_scala_binary(
     ],
 )
 
-genrule(
-    name = "assembler-test-dalf",
-    srcs = ["src/test/resource/Test.damllf"],
-    outs = ["Test.dalf"],
-    cmd = "$(location :encoder_binary) $(SRCS) $@ 1.dev",
+
+
+lf_since_deps = {
+    "1.0": [],
+    "1.1": ["1.0"],
+    "1.3": ["1.0", "1.1"],
+    "1.dev": ["1.0", "1.1", "1.3"]
+}
+
+[
+    [filegroup(
+         name = "lf_since_%s" % target,
+         srcs = [":lf_since_%s" %dep for dep in lf_since_deps[target]] +
+           glob(["src/test/lf/since_%s/**/*.lf" % target]),
+    )]
+    for target in lf_target_versions
+]
+
+
+[
+  [genrule(
```

```suggestion
  genrule(
```
Also `bazel run //:buildifier-fix` I'd wager.

---

fixed